src/platform/web/ui/session/leftpanel/LeftPanelView.js

```diff
@@ -71,7 +68,8 @@ export class LeftPanelView extends TemplateView {
                 }
             }
         ));
-        const utilitiesRow = t.div({className: "utilities"}, [
+        const utilitiesRow = t.form({name:"utilities-form"}, [ 
```

Comment:
probably only the filter field should be in the form, not the button links?

Reply:
`const utilitiesRow = t.form({
  name: "utilities-form",
  onSubmit: (event) => {
    event.preventDefault(); // Prevent form submission
    // Additional code logic if needed
  },
}, [
  // Form elements
]);
`


> By adding the onSubmit event listener to the form element and calling event.preventDefault(), you can prevent the form from submitting when the enter key is pressed within the filter field or any other input field inside the form.

```